pkg/clamav/scanner.go

```diff
@@ -31,19 +32,19 @@ func NewScanner(socket string) (api.Scanner, error) {
 	}
 	return &ClamScanner{
 		Socket: socket,
-		clamd: clamSession,
+		clamd:  clamSession,
```

Comment:
reminder that there should be gofmt travis check

Reply:
https://github.com/openshift/image-inspector/issues/74